### DISCUSSION TITLE
Where do I find the v10 change log that appeared in nvim?

### DISCUSSION THREAD

**Discussion by j-steinbach** [LazyVim/LazyVim#2828] [General]
In nvim there was a 'What is new in version 10.x' post, with a lot of interesting information. I closed it and can't find it anymore. How do I show it again? Is it the same as the changelog at https://github.com/LazyVim/LazyVim/releases/tag/v10.0.0 ?

**Comment by dpetka2001** [comment]
`<leader>L` (capital L) should do the trick.

**Comment by folke** [comment]
You can do `:=LazyVim.news.lazyvim()`

I should probably make this a bit more discoverable :)

It will show [this](https://github.com/LazyVim/LazyVim/blob/main/NEWS.md) file

**Comment by j-steinbach** [comment]
Awesome, ty! Exactly what I was looking for!
